### PR TITLE
Adding missed files from refactor to dev

### PR DESCRIPTION
**Pull request by JenMadiedo**
Issue Number: (Link to Github Issue or Azure Dev Ops Task/Story)

## Summary
Realized there were a set of files that weren't properly moved into their new folders during the restructure last night. Also added the CLI test project to the solution so it gets run with the test command

## Changes

- Moving files to appropriate directory

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [ ] Documentation Updates Made?
- [ ] This is not a breaking change. If it is, please describe it below.

## Is this a breaking change?
If yes, what workflow does this break? 

## Anything else?
Other comments, collaborators, etc.
This PR Closes Issue #

### CODE REVIEW

File: src/CarbonAware.CLI/test/CarbonAwareCLITests.cs [hunk 1]
```diff
@@ -1,10 +1,10 @@
 using NUnit.Framework;
 using Moq;
-using Microsoft.Extensions.Logging;
-using CarbonAware.Plugins.JsonReaderPlugin;
 using System;
 using System.IO;
 using CarbonAware.Aggregators.CarbonAware;
+using Castle.Core.Logging;
```
Comment (juzuluag) [inline src/CarbonAware.CLI/test/CarbonAwareCLITests.cs]: nit: is this needed?

File: src/CarbonAware.CLI/test/CarbonAwareCLITests.cs [hunk 2]
```diff
@@ -36,32 +36,33 @@ public void ParseCommandLineArguments_SetsStartTimeAndEndTimeWhenProvided()
...
-    {
-        string[] args = new string[] {};
-        var stringWriter = new StringWriter();
-        Console.SetOut(stringWriter);
-        new CarbonAwareCLI(args, It.IsAny<ICarbonAwareAggregator>());
-        StringAssert.Contains("Required option 'l, location' is missing.", stringWriter.ToString());
-    }
+    // Commenting out for now due to swallowed error
+    // [Test]
+    // public void ParseCommandLineArguments_ThrowsErrorWhenLocationNotProvided()
```
Comment (bderusha) [inline src/CarbonAware.CLI/test/CarbonAwareCLITests.cs]: Prefer: `[Ignore("Ignore for now due to swallowed error")]`  and leave this uncommented

https://docs.nunit.org/articles/nunit/writing-tests/attributes/ignore.html

File: src/CarbonAware.CLI/test/CarbonAwareCLITests.cs (hunk 1 above)
Comment (JenMadiedo) [reply src/CarbonAware.CLI/test/CarbonAwareCLITests.cs]: No, totally missed this was added

**Review by bderusha — APPROVED** [review]
Interested in why there is a new test failure.  I'm fine skipping the test for now, but we should have a plan for how to track resolving it in the longterm

**Review by juzuluag — APPROVED** [review]

File: src/CarbonAware.CLI/test/CarbonAwareCLITests.cs (hunk 2 above)
Comment (JenMadiedo) [reply src/CarbonAware.CLI/test/CarbonAwareCLITests.cs]: Made the change to ignore instead. I debugged for a bit today but couldn't figure out the issue, I reached out to @pritipath  who originally wrote it and she'll be taking a look to see if she knows what's going on. My guess from what I saw was that the exception that is usually thrown keeps getting swallowed, I'm guessing in the general exception handling code we added.

Comment (JenMadiedo) [reply src/CarbonAware.CLI/test/CarbonAwareCLITests.cs]: Also this failure isn't specific to this change. I noticed that the CLI Tests weren't part of the solution and thus weren't being run as part of `dotnet test` so they were actually out of date regardless, so not quite sure which PR may have caused them to break originally.

**Pull request merged**